lib/webrtc/rtcpeerconnection/safari.js

```diff
@@ -13,7 +13,7 @@ const updateTrackIdsToSSRCs = isUnifiedPlan
   : updatePlanBTrackIdsToSSRCs;
 
 class SafariRTCPeerConnection extends EventTarget {
-  constructor(configuration) {
+  constructor(configuration, options = {}) {
```

Comment:
I see that the Chrome RTCPeerConnection has a default value for `configuration` where this Safari one does not.